dataland-data-exporter/src/test/resources/csv/configs/transformation.config

```diff
@@ -0,0 +1,4 @@
+presentMapping=presentHeader
+notMapped=
```

Comment:
Was macht dieses File genau?

Reply:
Das ist ein mapping file für Testzwecke. Wird in den unit tests verwendet um einfache Testfälle abzudecken.

Reply:
Danke